src/grid/TariffProfileGridView.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Finance module for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-finance
+ * @package   hipanel-module-finance
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2017, HiQDev (http://hiqdev.com/)
+ */
+
+namespace hipanel\modules\finance\grid;
+
+use hipanel\helpers\Url;
+use hiqdev\yii2\menus\grid\MenuColumn;
+use hipanel\modules\finance\menus\ProfileActionsMenu;
+use hipanel\modules\finance\models\Tariff;
+use yii\helpers\Html;
+
+use Yii;
+
+class TariffProfileGridView extends \hipanel\grid\BoxedGridView
+{
+    public function columns()
+    {
+        return array_merge(parent::columns(), [
+            'name' => [
+                'class' => 'hipanel\grid\MainColumn',
+                'filterAttribute' => 'name_like',
+                'note' => null,
+                'value' => function ($model) {
```

Comment:
Старайся добавлять Type Hinting


```suggestion
                'value' => function (TariffProfile $model): string {
```